{{cookiecutter.__src_folder_name}}/src/django/project/settings.py

```diff
@@ -16,14 +16,11 @@
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
-
-# Quick-start development settings - unsuitable for production
```

Comment:
This would scare customers!

Reply:
We could still keep the deployment checklist if you'd like though. Or put that in the README.